### PR TITLE
docs: add docs for BEP153 stake migration

### PR DESCRIPTION
**Pull request by forcodedancing**

### CODE REVIEW

File: docs/bcfusion/developers/staking.md [hunk 1]
```diff
@@ -134,7 +134,105 @@ function claimBatch(address[] calldata operatorAddresses, uint256[] calldata req
 - `operatorAddress`: The operator addresses of the validatores.
 - `requestNumber`: The numbers of unbonding requests to claim from the validators.
 
+## FAQs
+
+### What are the functions/interfaces of each validator's credit contract?
+
+For each validator, there is a credit contract which will be automatically deployed when it is created.
```
Comment (unclezoro) [inline docs/bcfusion/developers/staking.md:141]: this did not explain the `functions` of credit contract

File: docs/bcfusion/developers/staking.md [hunk 2]
```diff
@@ -134,7 +134,105 @@ function claimBatch(address[] calldata operatorAddresses, uint256[] calldata req
...
+The credit contract is a BEP20 contract, and the ABI is the same
+as [Stake Credit contract](https://github.com/bnb-chain/bsc-genesis-contract/blob/master/abi/stakecredit.abi).
+
+Meanwhile, the conctract cannot be upgraded or changed by any validator operator.
+
+### How to get the shares/BNB amount for a delegator?
+
+For any specific validator, you can call the `balanceOf` function of the validator's creat contract to get the
```
Comment (unclezoro) [inline docs/bcfusion/developers/staking.md:149]: try to avoid using `you`

File: docs/bcfusion/developers/staking.md [hunk 3]
```diff
@@ -134,7 +134,105 @@ function claimBatch(address[] calldata operatorAddresses, uint256[] calldata req
...
+
+Please be noted that each validator will have its own APR/APY, and the staking system will auto compound the rewards.
+
+The reward is distributed to each validator's BNB pool at 00:00:00 UTC time every day. To calculate the APR/APY of a
+validator, you need to get the total pooled BNB amount and the crrospanding reward amount for the same day.
+
+The `StakeHub` contract provides the `getValidatorTotalPooledBNBRecord(address,uint256)(uint256)`
+and `getValidatorRewardRecord(address,uint256)(uint256)` for the purpose.
+
+The following code shows how to get the total pooled BNB amount and the crrospanding reward amount for a given time:
```
Comment (unclezoro) [inline docs/bcfusion/developers/staking.md:174]: why not change "shows how to get the total pooled BNB amount and the crrospanding reward amount for a given time" ==> `shows how to calculate the APY at a given day`?

File: docs/bcfusion/developers/staking.md [hunk 4]
```diff
@@ -134,7 +134,105 @@ function claimBatch(address[] calldata operatorAddresses, uint256[] calldata req
...
+
+// get the block time of a given block
+header, _ := p.client.GetBlockHeader(blockHeight)
+
+// calculate the index paramter to call following functions
+index := int64(header.Time) / interval.Int64()
+
+// get the total pooled BNB amount and the crrospanding reward amount for the given validator and index
+totalPooledBNB, _ := stakeHub.GetValidatorTotalPooledBNBRecord(nil, validatorOperatorAddress, index)
+reward, _ := stakeHub.GetValidatorRewardRecord(nil, validatorOperatorAddress, index)
```
Comment (unclezoro) [inline docs/bcfusion/developers/staking.md:193]: Add one more step to calculate the APY

File: docs/bcfusion/users/bep153-stake-migration.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# Stake Migration For BEP153
+
+The [BEP-153](https://github.com/bnb-chain/BEPs/blob/master/BEPs/BEP153.md) has been introduced as a native staking
+protocol onto BNB Smart Chain. With this BEP, individual or institution delegators can stake BNB to specified validators
```
Comment (unclezoro) [inline docs/bcfusion/users/bep153-stake-migration.md:4]: ==> `onto BNB Smart Chain before BC Fusion.`

File: docs/bcfusion/users/bep153-stake-migration.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+2. Wait for the unbounding period (7 days)
+
+3. Then you can delegate your BNB to the new staking system by following the [new staking guide](new-stake.md)
+
+## For LSD Protocal Projects
+
+For LSD protocol projects, you **MUST** finish your undelegations before
+the [Second Sunset Fork](https://www.bnbchain.org/en/bnb-chain-fusion) of Beacon Chain.
+In the Second Sunset Fork, all the delegations will automatically be returned to the delegator addressess.
+If your delegation address is not a EOA address, you will lose the control of your stakes and the fund will be lost.
```
Comment (unclezoro) [inline docs/bcfusion/users/bep153-stake-migration.md:53]: smart contract is still ok, but need to be a "payable smart contract".  

`a EOA` ==> `an EOA`

File: docs/bcfusion/developers/staking.md (hunk 2 above)
Comment (forcodedancing) [reply docs/bcfusion/developers/staking.md:149]: updated

File: docs/bcfusion/developers/staking.md (hunk 3 above)
Comment (forcodedancing) [reply docs/bcfusion/developers/staking.md:174]: fixed

File: docs/bcfusion/developers/staking.md (hunk 4 above)
Comment (forcodedancing) [reply docs/bcfusion/developers/staking.md:193]: sure

File: docs/bcfusion/users/bep153-stake-migration.md (hunk 5 above)
Comment (forcodedancing) [reply docs/bcfusion/users/bep153-stake-migration.md:4]: updated

File: docs/bcfusion/users/bep153-stake-migration.md (hunk 6 above)
Comment (forcodedancing) [reply docs/bcfusion/users/bep153-stake-migration.md:53]: fixed

File: docs/bcfusion/developers/staking.md (hunk 1 above)
Comment (forcodedancing) [reply docs/bcfusion/developers/staking.md:141]: updated

**Review by unclezoro — APPROVED** [review]

**Pull request merged**